### PR TITLE
Change typo and style

### PR DESCRIPTION
**Pull request by kencho51**
# Pull request for issue: #873 #874 

This is a pull request for the following functionalities:

<Insert description of the proposed new functionalities and how they can be tested>

## Changes to the view template files
1. Fixed the style  <del><em>GigaDB</em></del> GigaDB User policies`.
2. Fixed the typo in `Hosting Statement`. 

## Changes to the style
1. Removed the `<em></em>` from GigaDB User Policies heading.

## Changes to the tests
1. To test that `ll` has been changed to `All`.

### CODE REVIEW

**Review by rija — CHANGES_REQUESTED** [review]
Hi @kencho51,

fix looks ok

however:

* It is not necessary to write a test for change to the static text content (remember what a test suite is for: preserving and increasing the quality of the code, preventing regressions and act as a safety net to the developer when they work on the code). Eventually that text and the fixing of the typo will be handled with a CMS
* The change to styling of a tab title is navigational element, so it's part of the user interface, so it's front-end code, so it's code, so it needs a test

**Review by rija — APPROVED** [review]
Thanks @kencho51, 
tests are all passing locally and on CI and the fixes work on my staging deployment.
Happy to approve this PR

File: features/site-overall.feature [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Feature: A user visit gigadb website
+  As a gigadb user
+  All front end views display properly
```
Comment (pli888) [inline features/site-overall.feature:3]: If we want to be consistent with other feature descriptions then this needs correcting to: `I want to see all front end views displayed properly`

File: features/site-overall.feature [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  So that I can experience a good performance of gigadb website
+
+  Background:
+    Given Gigadb web site is loaded with production-like data
+
+  @ok @issue-883
```
Comment (pli888) [inline features/site-overall.feature:9]: I don't think we need the `@issue-883` tag. It's not a tag that I would expect to use in the future to execute a set of acceptance tests.

File: features/site-overall.feature [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    Scenario: Terms - GigaDB User Policies
+      Given I am not logged in to Gigadb web site
+      When I go to "/site/term"
+      Then I should see "GigaDB User Policies"
+      And I should not see "<em>GigaDB</em> User Policies"
```
Comment (pli888) [inline features/site-overall.feature:14]: Can you check the whitespace formatting? It seems odd compared to other features.

**Review by pli888 — APPROVED** [review]
Code looks good to me - just the cosmetic changes I suggested.

**Review by rija — CHANGES_REQUESTED** [review]
Hi @kencho51,

it slipped my attention when I looked at this PR last time,

would you mind correct the user story to follow the conventions we use in product backlogs and to be more specific?

In particular,  there is no such user as "gigadb user". Something like this would be better:

>As a website user
>I want to see useful and consistent navigational controls in the website's static pages area
>So that I can easily navigate the legal and help documentations

**Comment by kencho51** [comment]
Hi @rija ,
> 
> In particular, there is no such user as "gigadb user". Something like this would be better:
> 
> > As a website user
> > I want to see useful and consistent navigational controls in the website's static pages area
> > So that I can easily navigate the legal and help documentations

I would like to keep this line `So that I can experience a good performance of gigadb website` as this feature is intended to hold the tests for the overall view of the website, like the guideline pages, term of use page, etc.

**Comment by rija** [comment]
> Hi @rija ,
> 
> > In particular, there is no such user as "gigadb user". Something like this would be better:
> > > As a website user
> > > I want to see useful and consistent navigational controls in the website's static pages area
> > > So that I can easily navigate the legal and help documentations
> 
> I would like to keep this line `So that I can experience a good performance of gigadb website` as this feature is intended to hold the tests for the overall view of the website, like the guideline pages, term of use page, etc.

@kencho51,

The element you are listing are called static content. "Overall" means "Including everything; comprehensive." which is not what these issues are about.
The word "performances" has no place in here, as these are not performances test, we are not assessing how these pages are performing against a baseline.

Acutally, you should probably change the feature's file name to something like ``static-content-navigation.feature``

**Comment by kencho51** [comment]
Hi @rija 
> 
> The element you are listing are called static content. "Overall" means "Including everything; comprehensive." which is not what this issue is about since dynamic content is not in scope of this issue. The word "performance" has no place in here, as these are not performance tests, we are not assessing how these pages are performing against a baseline on some measurable criteria.
> 
> Acutally, you should also change the feature's file name to something like `static-content-navigation.feature`

After rebasing to the latest develop code, the acceptence test has been using codeception infrastructure and a new file `StaticContentNavigation.feature` has been created.
And the user story is updated too.

**Review by rija — APPROVED** [review]

**Comment by rija** [comment]
Thanks @kencho51, all good now!

**Pull request merged**